tests/queries/0_stateless/02350_views_max_insert_threads.sql

```diff
@@ -5,7 +5,7 @@ drop table if exists t_mv;
 create table t (a UInt64) Engine = Null;
 create materialized view t_mv Engine = Null AS select now() as ts, max(a) from t group by ts;
 
-insert into t select * from numbers_mt(10e6) settings max_threads = 16, max_insert_threads=16;
+insert into t select * from numbers_mt(10e6) settings max_threads = 16, max_insert_threads=16, max_block_size=50;
```

Comment:
Could you please explain this change?

Reply:
Because `StorageSystemNumbers` will adjust num_streams, for example: we need scan 100 rows and with settings max_threads=10 and max_block_size=50, then we will adjust real number of stream to 2.

Details see:
https://github.com/ClickHouse/ClickHouse/pull/50909/files#diff-2050c1a85be8c9e378b44fcb41942285db6225f28046ff368468d76b51e2e626R285-R286